src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -66,7 +66,7 @@ public RuntimeMirror(string varname, int blockDecl, ProtoCore.RuntimeCore runtim
 
                 variableName = varname;
                 blockDeclaration = blockDecl;
-                StackValue svData = deprecateThisMirror.GetValue(variableName, blockDeclaration).DsasmValue;
```

Comment:
If the value for this variable is an array, `GetValue()` will convert it to C# list. But we are only interested in its `StackValue`